sdk/eventhub/Microsoft.Azure.EventHubs/src/Primitives/EventHubsException.cs

```diff
@@ -10,6 +10,22 @@ namespace Microsoft.Azure.EventHubs
     /// </summary>
     public class EventHubsException : Exception
     {
+        /// <summary>
+        ///Enumerates the types of error level for the messaging communication.
+        /// </summary>
+        public enum ErrorLevelType
```

Comment:
nit: I'm on the fence as to whether this enum should stand alone as an entity or whether it only makes sense in the context of this particular class.   I suppose it really doesn't matter, since developers won't make use of it directly, outside referencing from an exception property.

Reply:
Only makes sense in the context of this class. 

---
In reply to: [311764484](https://github.com/Azure/azure-sdk-for-net/pull/7194#discussion_r311764484) [](ancestors = 311764484)